.circleci/workflows.yml

```diff
@@ -48,85 +80,110 @@ parameters:
-  docker-scan:
+  publish_bundle:
+    parameters:
+      dry_run:
+        type: boolean
+    steps:
+      - keeper/env-export:
+          secret-url: keeper://Mqmplmfu17bDR5XRLmO1mQ/field/password
+          var-name: AWS_ACCESS_KEY_ID
+      - keeper/env-export:
+          secret-url: keeper://3-pU56sIqcyWWw7HxhxjaQ/field/password
+          var-name: AWS_SECRET_ACCESS_KEY
+      - when:
+          condition: << parameters.dry_run>>
+          steps:
+            - aws-s3/sync:
+                arguments: |
+                  --endpoint-url https://cellar-c2.services.clever-cloud.com \
+                  --acl public-read
+                from: ~/release/
+                to: "s3://gravitee-dry-releases-downloads/"
+      - when:
+          condition:
+            not: << pipeline.parameters.dry_run>>
+          steps:
+            - aws-s3/sync:
+                arguments: |
+                  --endpoint-url https://cellar-c2.services.clever-cloud.com \
+                  --acl public-read
+                from: ~/release/
+                to: "s3://gravitee-releases-downloads/" 
+  restore-maven-job-cache:
+    description: Restore Maven cache for a dedicated job
+    parameters:
+      jobName:
+        description: The job name
+        type: string
+        default: ""
+    steps:
+      - restore_cache:
+          keys:
+            - gravitee-access-management-v1-<< parameters.jobName >>-{{ .Branch }}-{{ checksum "pom.xml" }}
+            - gravitee-access-management-v1-<< parameters.jobName >>-{{ .Branch }}-
+            - gravitee-access-management-v1-<< parameters.jobName >>-
+  save-maven-job-cache:
+    description: Restore Maven cache for a dedicated job
+    parameters:
+      jobName:
+        description: The job name
+        type: string
+        default: ""
+    steps:
+      - run:
+          name: "Exclude all AM artefacts from cache."
+          command: rm -rf ~/.m2/repository/io/gravitee/am
+      - save_cache:
+          paths:
+            - ~/.m2
+          key: gravitee-access-management-v1-<< parameters.jobName >>-{{ .Branch }}-{{ checksum "pom.xml" }}
+          when: always
+  notify-on-failure:
+    steps:
+      - keeper/env-export:
+          secret-url: keeper://ZOz4db245GNaETVwmPBk8w/field/password
+          var-name: SLACK_ACCESS_TOKEN
+      - slack/notify:
+          channel: C0336B2E96K
+          event: fail
+          template: basic_fail_1
+  notify-on-success:
     steps:
-      - snyk/scan:
-          organization: gravitee.io
-          monitor-on-build: false
-          severity-threshold: critical
-          docker-image-name: ${IMAGE_TAG}
-          fail-on-issues: false
-          additional-arguments: --app-vulns
+      - keeper/env-export:
+          secret-url: keeper://ZOz4db245GNaETVwmPBk8w/field/password
+          var-name: SLACK_ACCESS_TOKEN
+      - slack/notify:
+          channel: C0336B2E96K
+          event: pass
+          template: basic_success_1
+  add-docker-image-in-snyk:
```

Comment:
Not sure about this command...

The `docker-image-name `doesn't contains the organization and `graviteeio` is hard coded into the JSON payload.
But this command is used after the publication of the images after the ACR upload.

IHMO, we should make the organization as a parameter to be able to register the docker images into snyk after a publish on the Azure registry and after a release in DockerHub.

wydt ?